# ToFu shouldn't hardcode CC and CXX, they are set by environment.
# S.D.Pinches, 20190906
diff -Nru tofu-1.4.1.orig/setup.py tofu-1.4.1/setup.py
--- tofu-1.4.1.orig/setup.py	2019-09-04 09:45:51.000000000 +0200
+++ tofu-1.4.1/setup.py	2019-09-06 02:29:22.000000000 +0200
@@ -42,13 +42,13 @@
 if platform.system() == "Windows":
     is_platform_windows = True
 
-if platform.system() == "Darwin":
-    # make sure you are using Homebrew's compiler
-    os.environ['CC'] = 'gcc-8'
-    os.environ['CXX'] = 'g++-8'
-else:
-    os.environ['CC'] = 'gcc'
-    os.environ['CXX'] = 'g++'
+#if platform.system() == "Darwin":
+#    # make sure you are using Homebrew's compiler
+#    os.environ['CC'] = 'gcc-8'
+#    os.environ['CXX'] = 'g++-8'
+#else:
+#    os.environ['CC'] = 'gcc'
+#    os.environ['CXX'] = 'g++'
 
 # ==============================================================================
 class CleanCommand(Clean):
